src/config.ts

```diff
@@ -24,6 +24,11 @@ export const DEFAULT_METRIC_DEBOUNCE_TIME = {
   min: 1000,
   max: 60000,
 };
+export const DEFAULT_CACHE_PATH = joinPath(
+  process.cwd(),
+  '.cache',
+  'superface'
+);
```

Comment:
Would creating .cache under superface dir make sense? We would have everything under same folder 🤷

Reply:
i suppose. we talked about it, but i don't remember what we settled on. in that case we should update init to put a .gitignore in superface directory

Reply:
I think we just listed options 🤷😅 
Agree on .gitignore